types/node/fs.d.ts

```diff
@@ -2784,21 +2807,34 @@ declare module 'fs' {
      * * the file is renamed and then renamed a second time back to its original name
      * @since v0.1.31
      */
+    export interface WatchFileOptions {
+        bigint?: boolean | undefined;
+        persistent?: boolean | undefined;
+        interval?: number | undefined;
+    }
     export function watchFile(
         filename: PathLike,
         options:
-            | {
-                  persistent?: boolean | undefined;
-                  interval?: number | undefined;
-              }
+            | (WatchFileOptions & {
+                  bigint?: false | undefined;
+              })
             | undefined,
         listener: (curr: Stats, prev: Stats) => void
-    ): void;
+    ): StatWatcher;
+    export function watchFile(
+        filename: PathLike,
+        options:
+            | (WatchFileOptions & {
+                  bigint?: true;
```

Comment:
iirc `bigint` must be explicitly true here.

Reply:
Yes good catch, that was a typo on my part